meta-balena-common/classes/kernel-balena.bbclass

```diff
@@ -636,6 +636,17 @@ BALENA_CONFIGS[dmcrypt] = " \
     CONFIG_DM_CRYPT=y \
 "
 
+BALENA_CONFIGS_append = "${@oe.utils.conditional('SIGN_API','','','secureboot',d)}"
+BALENA_CONFIGS[secureboot] = " \
+    CONFIG_MODULE_SIG=y \
+    CONFIG_MODULE_SIG_ALL=y \
+    CONFIG_MODULE_SIG_FORCE=y \
+    CONFIG_MODULE_SIG_SHA512=y \
+    CONFIG_SECURITY_LOCKDOWN_LSM=y \
+    CONFIG_SECURITY_LOCKDOWN_LSM_EARLY=y \
+    CONFIG_SYSTEM_TRUSTED_KEYS="certs/kmod.crt" \
+"
```

Comment:
It seems that we can remove `CONFIG_MODULE_SIG_FORCE=y` and use the `module.sig_enforce` kernel command line parameter when wanting to enforce that only signed modules are loaded.

Reply:
About lockdown, just enabling `CONFIG_SECURITY_LOCKDOWN_LSM` and `CONFIG_SECURITY_LOCKDOWN_LSM_EARLY` uses the default integrity mode which is none. The `integrity` mode is then set in the kernel patch if securemode is detected. This can also be set via the command line.

Reply:
Done